docs/settings/logging-settings.asciidoc

```diff
@@ -4,172 +4,87 @@
-
-Log the default log format to JSON layout instead of pattern (the default).
-With `json` layout, log messages will be formatted as JSON strings in https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[ECS format] that includes a timestamp, log level, logger, message text and any other metadata that may be associated with the log message itself.
-
-[source,yaml]
-----
-logging:
-  appenders:
-    json-layout:
-      type: console
-      layout:
-        type: json
-  root:
-    appenders: [json-layout]
-----
-
-[[log-with-meta-to-stdout]]
-===== Log with meta to stdout
-
-Include `%meta` in your pattern layout:
-
-[source,yaml]
-----
-logging:
-  appenders:
-    console-meta:
-      type: console
-      layout:
-        type: pattern
-        pattern: "[%date] [%level] [%logger] [%meta] %message"
-  root:
-    appenders: [console-meta]
-----
-
-[[log-elasticsearch-queries]]
-===== Log {es} queries
-
-[source,yaml]
---
-logging:
-  appenders:
-    console_appender:
-      type: console
-      layout:
-        type: pattern
-        highlight: true
-  root:
-    appenders: [console_appender]
-    level: warn
-  loggers:
-    - name: elasticsearch.query
-      level: debug
---
-
-[[change-overall-log-level]]
-===== Change overall log level
-
-[source,yaml]
-----
-logging:
-  root:
-    level: debug
-----
-
-[[customize-specific-log-records]]
-===== Customize specific log records
-Here is a detailed configuration example that can be used to configure _loggers_, _appenders_ and _layouts_:
-
-[source,yaml]
-----
-logging:
-  appenders:
-    console:
-      type: console
-      layout:
-        type: pattern
-        highlight: true
-    file:
-      type: file
-      fileName: /var/log/kibana.log
-    custom:
-      type: console
-      layout:
-        type: pattern
-        pattern: "[%date][%level] %message"
-    json-file-appender:
-      type: file
-      fileName: /var/log/kibana-json.log
-      layout:
-        type: json
-
-  root:
-    appenders: [console, file]
-    level: error
-
-  loggers:
-    - name: plugins
-      appenders: [custom]
-      level: warn
-    - name: plugins.myPlugin
-      level: info
-    - name: server
-      level: fatal
-    - name: optimize
-      appenders: [console]
-    - name: telemetry
-      appenders: [json-file-appender]
-      level: all
-    - name: metrics.ops
-      appenders: [console]
-      level: debug
-----
-
-Here is what we get with the config above:
-[options="header"]
+The folowing table serves as a quick reference for different logging configuration keys. Note that these are not stand-alone settings and may require additional logging configuration. See the <<logging-configuration, Configure Logging in {kib}>> guide and complete <<log-settings-examples, examples>> for common configuration use cases. 
+
+[cols="2*<"]
 |===
+| `logging.appenders[].<appender-name>`
+| Unique appender identifier.
 
-| Context name | Appenders | Level
+| `logging.appenders[].console:`
```

Comment:
I used a similar format for indicating an array as maps used in [7.16](https://www.elastic.co/guide/en/kibana/7.16/settings.html), i.e. `map.regionmap.layers[].attribution`.